src/components/structures/RightPanel.js

```diff
@@ -34,7 +34,7 @@ import {Action} from "../../dispatcher/actions";
 export default class RightPanel extends React.Component {
     static get propTypes() {
         return {
-            roomId: PropTypes.string, // if showing panels for a given room, this is set
+            room: PropTypes.string, // if showing panels for a given room, this is set
```

Comment:
Shouldn't be an object then?

Reply:
Errr, yes, my bad. Should be an instanceOf(Room) if anything